build/conda-nonconda-test-requirements.txt

```diff
@@ -13,3 +13,5 @@ bqplot
 K3D
 ipyleaflet
 jinja2==3.0.3 # https://github.com/microsoft/vscode-jupyter/issues/9468#issuecomment-1078468039
+matplotlib
```

Comment:
Added tests for matplotlib, I believe this is a must as this could be one of the most popular widgets.